README.md

```diff
@@ -12,13 +12,22 @@ See https://istreamlabs.github.io/docs/sdks/golang
   - `.openapi-generator-ignore`
   - `convenience.go`
 
-A clean directory is required to ensure defunct files are removed. However, some like the above mentioned files do need to be kept to ensure the SDK is generated properly.
+A clean directory is required to ensure defunct files are removed. However, some like the above mentioned files do need to be kept to ensure the SDK is generated properly. 
```

Comment:
If I'm reading right, can we remove lines 11-13 also?

Reply:
Cleaned up docs in https://github.com/istreamlabs/go-sdk/pull/16/commits/5aa7466b436949a369205a2f2e3cf3c3b42a13e3